dbt_copilot_helper/commands/check_cloudformation.py

```diff
@@ -49,6 +50,25 @@ def lint(ctx: click.Context, directory: str) -> bool:
     return success
 
 
+@check_cloudformation.command()
+@click.option("-d", "--directory", type=str, default="copilot")
+@click.pass_context
+def static(ctx: click.Context, directory: str) -> bool:
```

Comment:
What does `static` mean in this context?

Reply:
Imagining a rename of `check-cloudformation` imminently and the whole thing, verb approach, maybe...

`copilot-helper cloudformation check-security`?